src/test/resources/capabilities-bstack-parallel-browsers.yml

```diff
@@ -0,0 +1,58 @@
+testEndpoint: https://bstackdemo.com
+
+driverType: remoteDriver
+
+localDriver:
```

Comment:
LocalDriver shouldn't be there for this profile?

Reply:
Removed Local Driver